test/e2e/accounts/common.ts

```diff
@@ -0,0 +1,340 @@
+import { privateToAddress } from 'ethereumjs-util';
+import FixtureBuilder from '../fixture-builder';
+import {
+  PRIVATE_KEY,
+  PRIVATE_KEY_TWO,
+  WINDOW_TITLES,
+  clickSignOnSignatureConfirmation,
+  convertETHToHexGwei,
+  switchToOrOpenDapp,
+  unlockWallet,
+  validateContractDetails,
+} from '../helpers';
+import { Driver } from '../webdriver/driver';
+
+export const TEST_SNAPS_SIMPLE_KEYRING_WEBSITE_URL =
+  'https://metamask.github.io/snap-simple-keyring/1.0.1/';
+
+export const ganacheOptions = {
+  accounts: [
+    {
+      secretKey: PRIVATE_KEY,
+      balance: convertETHToHexGwei(25),
+    },
+    {
+      secretKey: PRIVATE_KEY_TWO,
+      balance: convertETHToHexGwei(25),
+    },
+  ],
+};
+
+export const accountSnapFixtures = (title: string | undefined) => {
```

Comment:
Nit: I've found it helpful when test helpers are JSDoc'd, so I can understand what they're for and how they are used. It might be nice to do the same thing here.

Reply:
Added a JSDoc here